doc/content/devices/models/browan-tbms100/_index.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "Browan TBMS100"
+description: ""
+weight: 
+---
+
+{{< figure src="tbms100.png" alt="Generic Node Sensor Edition" class="float plain" >}}
+
+The Browan [TBMS100](https://www.browan.com/product/motion-sensor-pir/detail) is a PIR motion sensor designed to detect occupancy or movement in homes or buildings for security and efficient facility management. It helps optimize the use of spaces like desks, conference rooms, and bathrooms, ensuring better utilization.
+
+# Finding the Provisioning Information
+
+The provisioning information, such as **JoinEUI** and **DevEUI**, can be found on the back of the device. 
+
+{{< figure src="tbms100-back.png" alt="TBMS100 provisioning information" >}}
+
+{{< note "The **AppKey**, which is sensitive information, is provided within an Excel sheet." />}}
+
+# Onboarding to {{% tts %}}
+
+The TBMS100 doesn't support onboarding with a QR code. You must proceed directly to the options **Select the end device in the LoRaWAN Device Repository** or **Enter end device specifics manually**.
```

Comment:
please refer to existing documentation here, for example you can link the guide for onboarding using a QR code and other sections